app/models/concerns/webhooks/outgoing/endpoint_support.rb

```diff
@@ -27,6 +28,10 @@ def valid_event_types
     Webhooks::Outgoing::EventType.all
   end
 
+  def creative_concepts
+    Scaffolding::AbsolutelyAbstract::CreativeConcept.all
```

Comment:
We should scope this by the team the endpoint is being created on, otherwise we'll bleed info across tenants. 

```
team.scaffolding_absolutely_abstract_creative_concepts
```